config/routes.rb

```diff
@@ -2,14 +2,20 @@
   mount Blazer::Engine, at: "blazer"
 
   devise_scope :user do
+    # TODO: (possibly)
```

Comment:
Hm, I think you're right - feel free to refactor this appropriately, thanks!

Reply:
Had a go at this, but will leave it for now as it's confusing me a little. Also I think additional, separate controllers are needed for things like contributions. Might be worth making a task in Jira?